test/integration/test.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+'use strict';
+const path = require('path');
+const Listr = require('listr');
+const tempy = require('tempy');
+const execa = require('execa');
+const del = require('del');
+
+const packages = new Map([
+	['got', 'https://github.com/sindresorhus/got'],
+	['ava', 'https://github.com/avajs/ava']
```

Comment:
Some more:

- https://github.com/sindresorhus/pageres
- https://github.com/sindresorhus/np
- https://github.com/sindresorhus/ora
- https://github.com/sindresorhus/p-map
- https://github.com/sindresorhus/detect-indent
- https://github.com/sindresorhus/os-locale
- https://github.com/sindresorhus/execa
- https://github.com/sindresorhus/pify
- https://github.com/sindresorhus/boxen
- https://github.com/sindresorhus/make-dir


@kevva Can you list some of your larger repos?

@SamVerschueren You should add `listr` and some of your other repos.